addon/ng2/models/webpack-build-production.ts

```diff
@@ -15,7 +15,6 @@ export const getWebpackProdConfigPartial = function(projectRoot: string, appConf
     },
     plugins: [
       new WebpackMd5Hash(),
-      new webpack.optimize.DedupePlugin(),
       new webpack.optimize.UglifyJsPlugin(<any>{
         mangle: { screw_ie8 : true, keep_fnames: true },
```

Comment:
There was discussion in the core Angular project about a fix there that would remove the need for "keep_fnames: true" - I assume from this unchanged here, that such a fix did not make it in to RC6?